src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Mobile/MyExtensionsApp._1.Mobile.csproj

```diff
@@ -24,6 +24,11 @@
 		<!-- Required for C# Hot Reload -->
 		<UseInterpreter Condition="'$(Configuration)' == 'Debug' and $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'maccatalyst'">True</UseInterpreter>
 		<IsUnoHead>true</IsUnoHead>
+
+		<!--#if (mauiEmbedding)-->
+		<!-- Hot restart not working well with Maui Embedding -->
+		<UnoDisableHotRestartHelperGeneration>true</UnoDisableHotRestartHelperGeneration>
```

Comment:
Need to recheck this - I suspect this isn't required any more as the source generators check for reference to maui